common/cache/cache.go

```diff
@@ -53,6 +53,10 @@ type Cache interface {
 
 	// Size returns the number of entries currently stored in the Cache
 	Size() int
+
+	// EvictItemsPastTimeToLive evicts all items in the cache which are expired
+	// This is not called automatically, but can be called periodically to evict expired items
+	EvictItemsPastTimeToLive()
```

Comment:
doesn't need to be exposed

Reply:
Agree, this makes a lot of things simpler, thanks!